app/src/test/java/uk/ac/ebi/atlas/experimentimport/ExperimentCheckerIT.java

```diff
@@ -66,6 +68,8 @@ void checkAllFiles(String experimentAccession, ExperimentType experimentType) {
         return Stream.of(
                 RNASEQ_MRNA_BASELINE,
                 PROTEOMICS_BASELINE,
+                PROTEOMICS_DIFFERENTIAL,
+                PROTEOMICS_BASELINE_DIA,
```

Comment:
Thanks for Adding 'DIA' experiment type here

Reply:
I ran this class, one of the test cases is failing.
Here is the details: 
```
ExperimentCheckerIT > checkAllFiles(String, ExperimentType) > uk.ac.ebi.atlas.experimentimport.ExperimentCheckerIT.checkAllFiles(String, ExperimentType)[2] FAILED
    java.lang.IllegalStateException at ExperimentCheckerIT.java:64
```
Can you please run this class on your side?

Reply:
I think we should think about renaming this `ExperimentCheckerIT.java` to `ExpressionAtlasExperimentCheckerIT.java` as ExperimentChecker is an interface.